README-ja.md

```diff
@@ -4,6 +4,9 @@
 
 このリポジトリには、[KubernetesのWebサイトとドキュメント](https://kubernetes.io/)をビルドするために必要な全アセットが格納されています。貢献に興味を持っていただきありがとうございます！
 
+- [ドキュメントにコントリビュートする](#contributing-to-the-docs)
+- [READMEの翻訳](#localization-readmemds)
```

Comment:
```suggestion
- [ドキュメントに貢献する](#contributing-to-the-docs)
- [翻訳された`README.md`一覧](#localization-readmemds)
```
These are links to the headings in this page.
I think the link text would be better to match the heading text.

And, it is necessary to set the IDs at the headings to be accessible.
```
## ドキュメントに貢献する {#contributing-to-the-docs}
```
```
## 翻訳された`README.md`一覧 {#localization-readmemds}
```

Reply:
ah thank you, let me add ids